README.md

```diff
@@ -8,6 +8,10 @@ This repository contains examples of simple applications built using ChemDraw JS
 
 A simple application that uses webpack, TypeScript and React. Shows how to configure Webpack to load ChemDraw JS.
 
+### JavaScript/HTML5
```

Comment:
Shouldn't this be "Vanilla JavaScript"? The name here should match the name of the package.